packages/kms-local/src/SphereonKeyManagementSystem.ts

```diff
@@ -268,4 +268,8 @@ export class SphereonKeyManagementSystem extends KeyManagementSystem {
     const signer = new RSASigner(PEMToJwk(hexToPEM(publicKeyHex, 'public'), 'public'), { hashAlgorithm, scheme })
     return await signer.verify(data, signature)
   }
+
+  public async listKeys(): Promise<ManagedKeyInfo[]> {
+    return (await this.privateKeyStore.list({})).map((privateKey) => this.asSphereonManagedKeyInfo(privateKey))
```

Comment:
Never ever use the private key store. We store both public and private keys. The private keys store is only used for signing and encryption. Never us that for external facing methods!

Reply:
Initially, I found it unusual to work with the private key store for this. But the `asSphereonManagedKeyInfo` does the trick and strips the private key part from this

Reply:
pushed a commit that fails if the returned keys have the `privateKeyHex` property